### DISCUSSION TITLE
Question: Fix Wide Letter Spacing

### DISCUSSION THREAD

**Discussion by rjhilgefort** [kovidgoyal/kitty#4132] [Q&A]
I just setup kitty on PopOS after using it on MacOS for a while now. I'm using the same config and the same font family (Dank Mono). In my PopOS setup, the letter spacing has wide gaps and I can't figure out how to configure/fix it. I've googled around and looked through the issues and don't see a configuration option or any  way to fix it. FWIW, it renders fine in the the default terminal app for PopOS.

I've looked through these configs: https://sw.kovidgoyal.net/kitty/conf/#fonts

Any help would be greatly appreciated, thanks! 

![image](https://user-images.githubusercontent.com/2034823/137637274-24103dd7-5173-4393-9a84-a949c67614f5.png)

**Comment by page-down** [comment]
It doesn't look like the font you mentioned is being used. Run the following command to see what font is being used?

```shell
kitty --debug-font-fallback
```

You can also get the debug config details according to the approach explicitly described in the issue template.

After confirming that the font is configured correctly and the font file is loaded correctly, you can try using `adjust_column_width` to adjust the width.

https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.adjust_line_height
